azure_functions_worker/main.py

```diff
@@ -26,7 +25,19 @@ def parse_args():
                              'syslog, or a file path')
     parser.add_argument('--grpcMaxMessageLength', type=int,
                         dest='grpc_max_msg_len')
-    return parser.parse_args()
+    parser.add_argument('--functions-uri', dest='functions_uri', type=str,
+                        help='URI with IP Address and Port used to'
+                             ' connect to the Host via gRPC.')
+    parser.add_argument('--functions-request-id', dest='functions_request_id',
+                        type=str, help='Request ID used for gRPC communication '
+                                       'with the Host.')
+    parser.add_argument('--functions-worker-id',
+                        dest='functions_worker_id', type=str,
+                        help='Worker ID assigned to this language worker.')
+    parser.add_argument('--functions-grpc-max-message-length', type=int,
+                        dest='functions_grpc_max_msg_len',
+                        help='Max grpc message length for Functions')
+    return parser
```

Comment:
Just confirming, if the new args are not present, the parsing will still work for old arguments(--host, --port etc.). correct?

Reply:
It should. If the new args are not present, it should be defaulted to None and the old args will be used.
@Prashant, add a unit test for this too.